docs/sdk-and-tools/rest-api/ws-subscriptions.md

```diff
@@ -364,13 +362,245 @@ main().catch(console.error);
 
 ---
 
+## Fresh Data Subscriptions (Custom Streams)
+
+**Note:** These streams provide real-time data with **no duplicates**. You must provide at least one filter criterion in the payload.
```

Comment:
i would rephrase to:
These streams provide real-time data (with a small delay after the actions are committed on-chain) for specific criteria. You must provide at least one filter criteria when subscribing